docs/configuration/intra_node_data_path.md

```diff
@@ -183,3 +190,84 @@ On the other hand, CARET requires users to provide the following description if
 
 User have to fill callback name in `variable_passings`, `publishes`'s `callback_name`. `context_type` must be set as `callback_chain`.
 After editing, use path.verify() described in the beginning of this section to verify that it has been set correctly.
+
+### Manipulating the object
+
+CARET serves Python-based APIs to define intra-node data path.
+The following commands enable `Architecture file editing` done in the previous section to be implemented by python commands.
+
+All of the following code snippets can be executed after the `Architecture('type', 'file_path')` method loads an architecture object.
+The architecture object whose sub-objects are renamed is saved to a file as explained in [the previous page](./load_and_save.md#save).
+
+#### `use_latest_message`
+
+Using the following Python API to use `use_latest_message` on an architecture object. The examples below follow the previous section.
+
+- You can add callback to publisher with `insert_publisher_callback` function in `Architecture` class.
+
+  - As arguments, the target node name, publishing topic name and publisher callback name must be specified.
+
+- You can update `context_types` to `use_latest_message` between targeted subscription and publisher with `update_message_context` function in `Architecture` class.
+  - As arguments, the target node name, subscription topic name and publisher topic name must be specified.
+
+```python
+# arch is caret_analyze.architecture.architecture.Architecture-based object
+
+arch.insert_publisher_callback('/pong_node', '/pong', 'timer_callback_1')
+arch.update_message_context('/pong_node', '/ping', '/pong', 'use_latest_message')
+```
+
```

Comment:
You can also remove intra-node data path of `use_latest_message` in Architecture object.

```python
# arch is caret_analyze.architecture.architecture.Architecture-based object
arch.remove_publisher_callback('/pong_node', '/pong', 'timer_callback_1')
arch.update_message_context('/pong_node', '/ping', '/pong', 'UNDEFINED')
```

Reply:
Fixed it in [ec38045](https://github.com/tier4/CARET_doc/pull/192/commits/ec38045c6dc7894a7a618e76683ed0b16673f01f).